examples/permissioned-token/test/common.js

```diff
@@ -0,0 +1,325 @@
+const {
+  getProgram
+} = require('@algo-builder/algob');
+const { Runtime, AccountStore, types } = require('@algo-builder/runtime');
+
+const minBalance = 20e6; // 20 ALGOs
+
+const CLAWBACK = 'clawback.py';
+const CONTROLLER = 'controller.py';
+const PERMISSIONS = 'permissions.py';
+const CLEAR_STATE = 'clear_state_program.py';
+
+const ALICE_ADDRESS = 'EDXG4GGBEHFLNX6A7FGT3F6Z3TQGIU6WVVJNOXGYLVNTLWDOCEJJ35LWJY';
+
+let master = new AccountStore(10000e6);
+let alice, bob, elon;
+let runtime;
+let assetIndex;
+let lsig;
+let controllerAppID, permissionsAppId;
+
+let CLAWBACK_PROGRAM;
+let CONTROLLER_PROGRAM;
+let PERMISSIONS_PROGRAM;
+const CLEAR_STATE_PROGRAM = getProgram(CLEAR_STATE);
+
+const TRANSFER = 'str:transfer';
+
+// Sync Accounts
+function syncAccounts (runtime) {
+  master = runtime.getAccount(master.address);
+  alice = runtime.getAccount(alice.address);
+  bob = runtime.getAccount(bob.address);
+  elon = runtime.getAccount(elon.address);
+}
+
+// Opt-In account to ASA
+function optInToASA (runtime, address, assetIndex) {
+  runtime.optIntoASA(assetIndex, address, {});
+}
+
+// Opt-In address to Permissions SSC
+function optInToPermissions (runtime, address, permissionsAppId) {
+  runtime.optInToApp(address, permissionsAppId, {}, {});
+}
+
+// Issue some tokens to the passed account
+function issue (runtime, manager, acc, amount, controllerAppID, assetIndex, lsig) {
```

Comment:
Please add doc string here, manager: Asset manager, acc: ?, amount: Issue amount,.
maybe change `acc` to `receiver`.